src/services/StructuralMetadataUtils.js

```diff
@@ -68,35 +68,31 @@ export default class StructuralMetadataUtils {
    * @param {Float} duration - end time of the media file in Milliseconds
    */
   buildSMUI(allItems, duration) {
-    // Regex to match hh:mm:ss, mm:ss OR seconds(as a float)/minutes(as an int)
-    const regexHHMMSS = /^([0-9]*:[0-9]*:[0-9]*.[0-9]*)$/i;
+    // Regex to match hh:mm:ss.ms, mm:ss.ms OR minutes(as an int)
+    const regexHHMMSS = /^([0-9]*:[0-9]*(:|.)[0-9]*.[0-9]*)$/i;
```

Comment:
I think this one line change should be reverted. As is, it will match mm.ss.ms, which we don't want.

Reply:
It doesn't match `mm.ss.ms` format with this regex according to this, https://regex101.com/r/URiYI0/2/

Reply:
I should have been more specific. It will match hh:mm.ss.ms, which we don't want.

Reply:
Same here: We really don't want the `(:|.)`. It should just be `:`.